### DISCUSSION TITLE
Does it work with Jetstream?

### DISCUSSION THREAD

**Discussion by Ricardosgeral** [balajidharma/laravel-vue-admin-panel#22] [General]
Hi
Good work
Does it work with Jetstream instead of breeze?
Regards

**Comment by balajidharma** [comment]
Hi
Thank you
No, it builds by using **breeze + Inertia + Vue**

**Reply by zenepay** [reply]
I love this. Unfortunately, Breeze does not have Two Factor authentication, If this would work with Jetstream or Breeze has 2FA, this will be the best one.
